src/components/MultiHopTrade/TradeSlideTransition.tsx

```diff
@@ -1,13 +1,21 @@
+import { useMediaQuery } from '@chakra-ui/react'
+import { useMemo } from 'react'
 import type { SlideTransitionProps } from 'components/SlideTransition'
 import { SlideTransition } from 'components/SlideTransition'
+import { breakpoints } from 'theme/theme'
 
-export const transitionStyle = {
-  width: '100%',
-  display: 'flex',
-  alignItems: 'center',
-  justifyContent: 'center',
-}
+export const TradeSlideTransition = (props: SlideTransitionProps) => {
+  const [isLargerThanMd] = useMediaQuery(`(min-width: ${breakpoints['md']})`, { ssr: false })
 
-export const TradeSlideTransition = (props: SlideTransitionProps) => (
-  <SlideTransition style={transitionStyle} {...props} />
-)
+  const style = useMemo(() => {
+    return {
+      width: '100%',
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      zIndex: 1400,
```

Comment:
We can't do this unless we use a higher z-index for the consent banner https://v2.chakra-ui.com/docs/styled-system/theme#z-index-values